src/ORM/Locator/TableLocator.php

```diff
@@ -274,7 +274,7 @@ public function exists(string $alias): bool
     /**
      * @inheritDoc
      */
-    public function set(string $alias, Table $object): Table
+    public function set(string $alias, $object): Table
```

Comment:
Could this be `object` ?

Reply:
Yes, `object` typehint can be used that argument here (and in the interfaces).